src/legacy/core_plugins/kibana/migrations.js

```diff
@@ -145,6 +145,38 @@ export const migrations = {
       } catch (e) {
         throw new Error(`Failure attempting to migrate saved object '${doc.attributes.title}' - ${e}`);
       }
+    },
+    '7.1.0': (doc) => {
+      // [TSVB] Migrate percentile-rank aggregation (value -> values)
+      const visStateJSON = get(doc, 'attributes.visState');
+
+      if (visStateJSON) {
+        let visState;
+        try {
+          visState = JSON.parse(visStateJSON);
+        } catch (e) {
+          // Let it go, the data is invalid and we'll leave it as is
+        }
+        if (visState && visState.type === 'metrics') {
+          const series = get(visState, 'params.series') || [];
+
+          series.forEach(series => {
```

Comment:
You're shadowing the series variable here: `series.forEach(series` maybe should become `series.forEach(s` or something.